src/mpyl/utilities/dagster/__init__.py

```diff
@@ -28,6 +29,7 @@ def from_dict(config: Dict):
                 global_service_account_override=dagster_config.get(
                     "globalServiceAccountOverride", None
                 ),
+                user_code_helm_chart_version=dagster_config["userCodeHelmChartVersion"],
```

Comment:
Maybe add a default version for old tags that don't have this value yet in the config